Update strings.xml_Chinese

jindouyunz opened the pull request:
updated on 4/22/2019 version 2, Jindouyunz.

jindouyunz committed:
Update strings.xml

updated on 4/22/2019 version 2, Jindouyunz.

erciccione commented:
Pinging past translators for review: @uiharu-s @bitOMG

erciccione commented:
Reping translators (Only the last 20-25 strings need to be reviewed)

Pinging some translators who helped with the Monero GUI wallet: @Equim-chan @4broaf @hqwrong

Please also review #573 and #574 if you can :)

erciccione commented:
Maybe @xmrnoblesse?

m2049r commented:
i will wait for review?

erciccione commented:
@m2049r I'm out of people to ping, if you want to push this, fine for me.

Lafudoci commented:
I'll review it now. Though I'm not simplified zh native speaker, but I could read most of them.

Lafudoci commented on app/src/main/res/values-zh-rCN/strings.xml:
```diff
@@ -1,382 +1,351 @@
+<string name="changepw_success">密码已修改</string>
+<string name="label_daemon">节点</string>
+<string name="connect_stagenet" translatable="false">Stagenet（供普通用户使用的门罗测试网）</string>
+<string name="connect_testnet" translatable="false">测试网（供开发用户使用的门罗测试网）</string>
```
Strings with attrib translatable="false" shouldn't be translated.

Lafudoci commented on app/src/main/res/values-zh-rCN/strings.xml:
```diff
@@ -1,382 +1,351 @@
+<string name="status_wallet_connect_ioex">无法连接至节点\n请重新尝试或连接其它的节点</string>
+<string name="status_wallet_disconnected">已断开连接</string>
+<string name="status_transaction_failed">交易失败: %1$s</string>
+<string name="send_xmrto_timeout">您已经等待足够长的时间了！</string>
```
Please check if this translation is accurate. Timeout means "too" long. But the translation seems to mean "enough".

Lafudoci commented on app/src/main/res/values-zh-rCN/strings.xml:
```diff
@@ -1,382 +1,351 @@
+<string name="prompt_fingerprint_auth">您也可以选择指纹登录。\n请触摸传感器。</string>
+<string name="prompt_send_password">确认密码</string>
+<string name="prompt_open_wallet">钱包打开中&#8230;</string>
+<string name="bad_fingerprint">无法是被您的指纹，请重试。</string>
```
"是被" should be a typo of "识别".

jindouyunz replied:
Much thanks Lafudoci，I'll revise those translations.

Lafudoci commented on app/src/main/res/values-zh-rCN/strings.xml:
```diff
@@ -1,382 +1,351 @@
+]]></string>
+<string name="generate_bad_passwordB">密码不匹配</string>
+<string name="generate_empty_passwordB">密码请不要为空</string>
+<string name="generate_buttonGenerate">Make me a wallet already!</string>
```
This string wasn't translated. From previous version it could be "建立钱包！".

Lafudoci commented on app/src/main/res/values-zh-rCN/strings.xml:
```diff
@@ -1,382 +1,351 @@
+<string name="generate_wallet_type_new">新</string>
+<string name="generate_wallet_type_seed">种子密语</string>
+<string name="generate_wallet_type_view">查阅</string>
+<string name="generate_wallet_type_ledger" translatable="false">Ledger</string>
```
Strings with attrib translatable="false" shouldn't be translated.

Lafudoci commented on app/src/main/res/values-zh-rCN/strings.xml:
```diff
@@ -1,382 +1,351 @@
+<string name="send_amount_hint">0.00</string>
+<string name="send_notes_hint">私密备注 (选填)</string>
+<string name="send_generate_paymentid_hint">生成</string>
+<string name="send_qr_hint" translatable="false">扫描</string>
```
Strings with attrib translatable="false" shouldn't be translated.

Lafudoci commented on app/src/main/res/values-zh-rCN/strings.xml:
```diff
@@ -1,382 +1,351 @@
+<item>THB</item>
+<item>TRY</item>
+<item>TWD</item>
+</string-array>
```
@m2049r Does this array should be here?

Lafudoci commented on app/src/main/res/values-zh-rCN/strings.xml:
```diff
@@ -1,382 +1,351 @@
+<string name="open_wallet_ledger_missing">请（重新）连接Ledger设备</string>
+<string name="accounts_progress_new">账户创建中</string>
+<string name="toast_ledger_attached">%1$s 已附加</string>
+<string name="toast_ledger_detached">%1$s 已从附加中删除</string>
```
Please check if these translation are accurate. The "attach" and "detach" means connect or disconnect to ledger device.

Lafudoci commented on app/src/main/res/values-zh-rCN/strings.xml:
```diff
@@ -1,382 +1,351 @@
+<string name="accounts_progress_new">账户创建中</string>
+<string name="toast_ledger_attached">%1$s 已附加</string>
+<string name="toast_ledger_detached">%1$s 已从附加中删除</string>
+<string name="progress_nfc_write">标签写入中</string>
```
Does "标签" enough to present "NFC label" well? "NFC标签" might be more precise.

Lafudoci commented on app/src/main/res/values-zh-rCN/strings.xml:
```diff
@@ -1,382 +1,351 @@
+<string name="fab_restore_key">通过私钥恢复钱包</string>
+<string name="fab_restore_seed">通过25词助记种子密语恢复钱包</string>
+<string name="accounts_balance" translatable="false">%1$s XMR</string>
+<string name="tx_account_formatted" translatable="false">(%1$d, %2$d)</string>
```
Strings with attrib translatable="false" shouldn't be translated.

Lafudoci commented on app/src/main/res/values-zh-rCN/strings.xml:
```diff
@@ -1,382 +1,351 @@
+<string name="nfc_tag_read_what">我不知道你想要什么！</string>
+<string name="nfc_tag_read_success">标签读取成功</string>
+<string name="nfc_tag_tap">NFC可用！</string>
+<string name="unknown_amount" translatable="false">\?\?\?</string>
```
Strings with attrib translatable="false" shouldn't be translated.

Lafudoci commented on app/src/main/res/values-zh-rCN/strings.xml:
```diff
@@ -1,382 +1,351 @@
+<string name="xmrto_error_010">无效或过期的支付URL</string>
+<string name="xmrto_error_012">当前请求数过多</string>
+<string name="xmrto_error" translatable="false">XMR.TO服务发生错误</string>
+<string name="street_sweep_amount">所有东西！</string>
```
Please check if these translation are accurate. This string means all the money in this account.

Lafudoci reviewed (changes requested):
Some translations should be revised.

jindouyunz committed:
Update string.xml_Chinese 2019-5-12

Hi, I made several pieces of modification based on lafudoci's review, thanks.

Lafudoci commented:
@m2049r @erciccione , IIRC, the strings with attrib translatable="false" could be removed from xml file, right? And the currency item array I don't know if it should be there.

erciccione commented:
Afaik, yes, they can be removed. 

That array is labelled `translatable= 'false'`, so i guess it could be removed, but i would wait @m2049r 's confirmation on that.

m2049r commented:
yes, `translatable=false` should be left out entirely - leaving them in creates errors on build.

Lafudoci commented:
@jindouyunz , sorry I should ask earlier. As you can see above comments, please remove all the string (entire line) with `translatable=false`, including the currency array. Then is PR should be good to go!

jindouyunz commented:
> 
> 
> @jindouyunz , sorry I should ask earlier. As you can see above comments, please remove all the string (entire line) with `translatable=false`, including the currency array. Then is PR should be good to go!

Sure，I'll do it now.

jindouyunz committed:
Update string.xml-zh-rCN 2019-5-13

Hi, I've removed all lines with "translatable=false", and currency array (<item> XMR<item> ……to <item>TWD</item>
</string-array>). Am I doing in an approach way?

Lafudoci reviewed (approved):
Translation reviewed.

erciccione reviewed (approved):
Looks good. Thanks to both :)
